gauntlet/packages/gauntlet-solana-contracts/src/lib/random.ts

```diff
@@ -0,0 +1,65 @@
+import * as readline from 'readline'
+import * as fs from 'fs'
+import { join } from 'path'
+
+export async function generateSecretWords(sampleSize: number = 12): Promise<string> {
```

Comment:
This method should probably be moved from `/lib/random` to some more appropriate location, somewhere in the OCR2 context.

Reply:
Since it's only used for `SharedSecretEncryptions` I've moved it [there]( https://github.com/smartcontractkit/chainlink-solana/pull/12/files#diff-5b625139ad869023583e0065d15d9200ea065b3366e133b1d168ea7f3d409c28R20).

`sample` function is still in `/lib/random`